react-app/package.json

```diff
@@ -64,6 +72,17 @@
     "stylelint-scss": "^4.2.0",
     "tailwindcss": "^3.0.24"
   },
+  "resolutions": {
+    "**/@cosmjs/amino": "0.28.4",
+    "**/@cosmjs/crypto": "0.28.4",
+    "**/@cosmjs/encoding": "0.28.4",
+    "**/@cosmjs/math": "0.28.4",
+    "**/@cosmjs/proto-signing": "0.28.4",
+    "**/@cosmjs/stargate": "0.28.4",
+    "**/@cosmjs/stream": "0.28.4",
+    "**/@cosmjs/tendermint-rpc": "0.28.4",
+    "**/@cosmjs/utils": "0.28.4"
+  },
```

Comment:
do we still need resolutions? `0.28.4` should be pass `^0.28.1`?

Reply:
Seems no, removed 🙇‍♂️